testsuite/functional-testsuite/src/test/ft-resources/testsuite-oss/support/support_logging.t.js

```diff
@@ -0,0 +1,122 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+/**
+ * Tests support logging.
+ */
+StartTest(function(t) {
+  var newLoggerName = 'aaa.example',
```

Comment:
I extracted this out to a method awhile back for convenience 
https://github.com/sonatype/nexus-oss/blob/master/testsupport/nexus-siestajs-testsupport/src/main/resources/ft-overlay/testsuite-lib/NX/TestClass.js#L234